contracts/accounts/base_account/Cargo.toml

```diff
@@ -22,8 +22,7 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 
 [dependencies]
 cosmwasm-schema       = { workspace = true }
-cosmwasm-std          = { workspace = true }
-
+cosmwasm-std          = { workspace = true, features = ["cosmwasm_1_4","stargate"] }
```

Comment:
Any reason we are adding cosmwasm1_4? If there isn't let's remove it or we won't be able to deploy on current mainnet Osmosis because they don't have this feature activated (due to devs messing up). But in general we shouldn't have features there we don't need.

Reply:
We are targetting the `cosmwasm1_4` feature of `cosmwasm_std` in the Workspace's cargo.toml.

Reply:
Yeah I think Ben removed it in his PR because it was giving trouble on Osmosis and we realized it wasn't being used anywhere

Reply:
ah ok